src/test/java/in/zapr/druid/druidry/query/aggregation/GroupByTest.java

```diff
@@ -18,25 +18,6 @@
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
-
```

Comment:
Too many unnecessary git diffs due to code rearrangement in many classes, can you fix these. (If you use repo style guide then this should be fixed)